speculos/client.py

```diff
@@ -82,6 +82,10 @@ def close_stream(self) -> None:
             self.stream.close()
         self.stream = None
 
+    def get_screen_content(self) -> dict:
+        resp = requests.get(f"{self.api_url}/events?screencontent=true")
```

Comment:
`screencontent` is ambiguous as it does not reflect it is an exclusive option (it filters the other events). Maybe `current_screen_only`?